ansible-playbook.yml

```diff
@@ -0,0 +1,62 @@
+- name: Install To-Do App on new web server
+  hosts: todo-app-group
+  remote_user: ec2-user
+  vars_prompt:
+  - name: api_key
+    prompt: api key?
+  - name: api_token
+    prompt: api token?
+  - name: organisation_id
+    prompt: organisation id?
+  - name: board_id
+    prompt: board id?
+
+
+  tasks:
+  - name: Install Git and Python 3
+    ansible.builtin.yum:
+      name: 
+      - git
+      - python
+      state: present
+    become: yes
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+      creates: ~/.poetry/bin/poetry
+
+  - name: Create To-Do App directory
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      mode: '777'
+    become: yes
+
+  - name: Get latest To-Do App from Git
+    ansible.builtin.git:
+      repo: https://github.com/KrisGwynne/DevOps-Course-Starter
+      dest: /opt/todoapp
+      version: exercise-4
```

Comment:
Usually it'd be nice to move this back to `main` before merging, but admittedly here we won't worry about keeping our Ansible up to date with future changes so feel free to leave this as is